ads_plugin.py

```diff
@@ -529,7 +524,7 @@ def on_save_segmentation_button(self, event):
         axon_array = axon_array * params.intensity['binary']
 
 
-        image_name = myelin_mask_overlay.name[:-len("_seg-myelin")]
+        image_name = myelin_mask_overlay.name.split("-myelin")[0]
```

Comment:
```suggestion
        image_name = myelin_mask_overlay.name.split("_seg")[0]
```
For the current naming convention in this PR, we have to change to split at `_seg` otherwise the name of the files would have duplicated `_seg_seg`.
Ex:
![fsleyes-wrong-filename](https://user-images.githubusercontent.com/54086142/150016019-056081eb-9e81-4863-8525-0f9d41de5319.png)
Overlay names in the GUI are OK:
![fsleyes_overlay_name](https://user-images.githubusercontent.com/54086142/150016046-d0ec624f-2562-439b-b94d-b750330e7052.png)


I suggest to not do the change for now, because I'll have some other news about suffixes that could change our strategy.

Reply:
Ok, on hold